lxc/image.go

```diff
@@ -1394,6 +1394,7 @@ func (c *cmdImageList) run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	images := make([]api.Image, len(allImages))
```

Comment:
```suggestion
	images := make([]api.Image, 0, len(allImages))
```

Same rational re length.